### DISCUSSION TITLE
Any plans to support Linux?

### DISCUSSION THREAD

**Discussion by breathiness** [mulaRahul/keyviz#22] [Q&A] [answered]
Are there plans to support Linux systems? I really like this software. Would like to use this on Linux as well.

---

有计划支持Linux系统吗？我非常喜欢这个软件。想在Linux上也使用这个。

**Comment by mulaRahul** [comment, marked as answer]
It **is** planned to add support for Linux and Mac in the future!

As mentioned in the #12 discussion, currently the software relies on a different dependency for keyboard input that supports only Windows. I'll have to write (or find) a suitable _plugin_ to read keyboard input for cross-platform support. _Alternatively, you may try [wine](https://www.winehq.org/) out_.

**Reply by breathiness** [reply]
wine is not supported, the bottom half of the screen turns black after installation and opening. 😂

![Snipaste_2022-08-17_00-11-25](https://user-images.githubusercontent.com/11361630/184930303-4ca8d9f6-aab2-439e-af56-79f116508b25.png)

**Reply by mulaRahul** [reply]
Guess it can't render the window transparent 😅. This calls for a native solution! (_the application relies on windows acrylic to do the same so it explains why the above happens_)

**Reply by mulaRahul** [reply]
@breathiness, you can checkout the [v2.0.0a](https://github.com/mulaRahul/keyviz/releases/tag/v2.0.0a) now!

**Comment by GT-610** [comment]
v2.0.0a does support Linux, but it shows a black screen. Anyway it's not usable now.

**Reply by mulaRahul** [reply]
Alright, will need to test the same.
